[Editorial review] BiDi - Add pages for browsingContext module, part 2: context lifecycle command and events (#44279)
mdn/content: files/en-us/web/webdriver/reference/bidi/modules/input/filedialogopened/index.md

### Receiving an event when a file picker dialog opens

> With a [WebDriver BiDi connection](/en-US/docs/Web/WebDriver/How_to/Create_BiDi_connection), an [active session](/en-US/docs/Web/WebDriver/Reference/BiDi/Modules/session/new), and a [subscription](/en-US/docs/Web/WebDriver/Reference/BiDi/Modules/session/subscribe) to `input.fileDialogOpened` active, consider a scenario where a page has an [`<input type="file">`](/en-US/docs/Web/HTML/Reference/Elements/input/file) element that accepts a single file and your script calls [`click()`](/en-US/docs/Web/API/HTMLElement/click) on it. The browser sends the following notification when the file picker dialog opens:

Reviewer: ```suggestion
Consider a sample app scenario with:

- A [WebDriver BiDi connection](/en-US/docs/Web/WebDriver/How_to/Create_BiDi_connection), an [active session](/en-US/docs/Web/WebDriver/Reference/BiDi/Modules/session/new), and an active [subscription](/en-US/docs/Web/WebDriver/Reference/BiDi/Modules/session/subscribe) to `input.fileDialogOpened` active.
- An [`<input type="file">`](/en-US/docs/Web/HTML/Reference/Elements/input/file) element that accepts a single file.
- A [`click()`](/en-US/docs/Web/API/HTMLElement/click) method called on the `<input>`.
 
The browser sends the following notification when the file picker dialog opens:
```

Maybe? The paragraph structure as-is feels a bit strange.

Author: I have structured this slightly differently because I'm using this pattern across other examples in BiDi.

Hopefully the flow is better with this:

> Assume you have a [WebDriver BiDi connection](/en-US/docs/Web/WebDriver/How_to/Create_BiDi_connection), an [active session](/en-US/docs/Web/WebDriver/Reference/BiDi/Modules/session/new), and a [subscription](/en-US/docs/Web/WebDriver/Reference/BiDi/Modules/session/subscribe) to `input.fileDialogOpened` active.
>
> Suppose your page has an [`<input type="file">`](/en-US/docs/Web/HTML/Reference/Elements/input/file) element that accepts a single file and your script calls [`click()`](/en-US/docs/Web/API/HTMLElement/click) on the `<input>`. The browser sends the following notification when the file picker dialog opens:

I have similarly redone the example setup prose in other files in this PR.

Reviewer: Yup, much better!
